File: solutions/0169-majority-element/solution.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var majorityElement = function(nums) {
    let dig = 0;
    let count = 0;
    for(let num of nums){
      if(count === 0){
        dig = num;
      }
      if(dig === num){
        count++;
      }
      else{
        count--;
      }

    }
    return dig;
};

File: solutions/0169-majority-element/solution.cs
public class Solution {
    public int MajorityElement(int[] nums) {
int dig = 0;
int count = 0;
for(int i = 0;i<nums.Length;i++){
    if(count == 0){
        dig = nums[i];
    }
    if(dig == nums[i]){
        count++;
    }
    else{
        count--;
    }
}
return dig;
    }
}
